profile.d/npm.sh

```diff
@@ -0,0 +1,22 @@
+export PATH="$HOME/.npm-packages/bin:$PATH" # man 1 sh
```

Comment:
Anything installed with `npm`/`yarn` will override everything else. Is it desireable?